### ISSUE TITLE
Support pulseaudio

### ISSUE THREAD

**Issue by janvda** [johanneskropf/node-red-contrib-sox-utils#3]
This is a kind request to also support pulseaudio.

I know that `arecord` normally requires ALSA, but using the alpine package `alsa-plugins-pulse` it is possible to use alsa clients (like sox, arecord, aplay) over pulseaudio.

FYI this is the message the node is reporting:

![image](https://user-images.githubusercontent.com/17238398/101277435-ca251180-37b4-11eb-81e3-2fb8e8696b0b.png)

I see that this node is using the below command to find the device list:
```
bash-5.0$ arecord -l
arecord: device_list:272: no soundcards found...
bash-5.0$ 
````

FYI the command `arecord -L` returns:

```
bash-5.0$ arecord -L
null
    Discard all samples (playback) or generate zero samples (capture)
default
    Default ALSA Output (currently PulseAudio Sound Server)
bash-5.0$ 
```

**Comment by janvda** [comment]
I managed to get it working for pulseaudio.
See the changes in my forked branch : https://github.com/janvda/node-red-contrib-sox-utils/tree/pulseaudio_support

The main change is that in case of error due to `arecord -l` : not to hide the options so that I can choose the `default` device and specify the options I want.

**Comment by johanneskropf** [comment]
Partially fixed in 0.4.1 see continuing discussion in https://github.com/johanneskropf/node-red-contrib-sox-utils/issues/4

**Comment by janvda** [comment]
I suggest to check my comment https://github.com/johanneskropf/node-red-contrib-sox-utils/issues/4#issuecomment-740610793


The default device setting is logging a warning.
